src/stripe.node.ts

```diff
@@ -0,0 +1,25 @@
+import Stripe = require('./stripe.common');
+import NodePlatformFunctions = require('./platform/NodePlatformFunctions');
+import http = require('http');
+import HttpClient = require('./net/HttpClient');
+
+Stripe._platformFunctions = new NodePlatformFunctions();
+
+Stripe.createNodeHttpClient = (agent: http.Agent): typeof HttpClient => {
```

Comment:
I'm hesitant to do this since I thought we previously exposed `createFetchHttpClient` and `createNodeHttpClient` as public APIs to be passed into the config as `httpClient`.